extensions/amp-bind/amp-bind.md

```diff
@@ -0,0 +1,207 @@
+# <a name="amp-bind-"></a> `amp-bind`
+
+**This extension is under active development, and the version number of the specification section should provide guidance to its evolution.**
+
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<table>
+  <tr>
+    <td class="col-fourty"><strong>Description</strong></td>
+    <td>`amp-bind` allows elements to mutate in response to user actions or data changes via data binding and simple JS-like expressions.</td>
```

Comment:
Github editor rendered what I wrote.  Please replace the backticks with `<code></code>`